lib/point_quest_web/live/quest.ex

```diff
@@ -22,15 +22,19 @@ defmodule PointQuestWeb.QuestLive do
           <.button phx-click="copy_link">Copy Invite Link</.button>
         </div>
       </div>
-
-      <div>
-        <pre><code><%= Jason.encode!(Ecto.embedded_dump(@quest, :json), pretty: true) %></code></pre>
-      </div>
       <div class="flex gap-4">
-        <div :for={{user_id, %{name: name, class: class}} <- @users} class="bg-blue-400">
-          <%= user_id %>
-          <%= name %>
-          <%= class %>
+        <!-- <div :for={{user_id, %{name: name, class: class}} <- @users} class="flex flex-col p-2"> -->
+        <div :for={%{id: id, class: class, name: name} <- @adventurers} class="flex flex-col p-2">
+          <div class={"#{get_background_color(id, @users)} rounded p-2"}>
+            <div id="class-sprite" class="w-[64px] h-[64px] mt-2">
```

Comment:
`w-[64px]` is the same as `w-16` and we can save a custom tailwind class.

Reply:
nice! will update shortly